jsk_spot_robot/jsk_spot_startup/package.xml

```diff
@@ -47,6 +47,8 @@
   <exec_depend>xacro</exec_depend>
   <exec_depend>xterm</exec_depend>
 
+  <exec_depend>esp_now_ros</exec_depend>
```

Comment:
please consder put `esp_now_ros` to jsk_3rdparty, for better dependency/mentainance